src/Test/Inspection/Plugin.hs

```diff
@@ -26,6 +26,7 @@ import Test.Inspection.Core
 --
 -- * @-fplugin-opt=Test.Inspection.Plugin:keep-going@ to ignore a failing build
 -- * @-fplugin-opt=Test.Inspection.Plugin:keep-going-O0@ to ignore a failing build when optimisations are off
+-- * @-fplugin-opt=Test.Inspection.Plugin:skip-O0@ to skip inspections when optimisations are off
 -- * @-fplugin-opt=Test.Inspection.Plugin:quiet@ to be silent if all obligations are fulfilled
```

Comment:
Can you rephrase the documentation to make it clear that `quiet` is orthogonal to the other three, and of the the other  three, at most one should be given?

Reply:
I revised the docs.